internal/model/ooapi.go

```diff
@@ -85,8 +85,8 @@ type OOAPICheckInResult struct {
 
 // OOAPICheckInResultConfig contains configuration.
 type OOAPICheckInResultConfig struct {
-	// Feature contains feature flags.
-	Feature map[string]bool `json:"feature"`
+	// Features contains feature flags.
+	Features map[string]bool `json:"features"`
```

Comment:
This was a typo that we didn't notice because we have not completed https://github.com/ooni/probe/issues/2372 yet.